app-i18n/man-pages-zh_CN/man-pages-zh_CN-1.6.3.2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A somewhat comprehensive collection of Chinese Linux man pages"
+HOMEPAGE="https://github.com/man-pages-zh/manpages-zh"
+MY_PN="manpages-zh"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="https://github.com/${PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
This URI seems to be wrong:

```
>>> Downloading 'https://github.com/man-pages-zh_CN/manpages-zh/archive/v1.6.3.2.tar.gz'
--2018-04-23 16:36:22--  https://github.com/man-pages-zh_CN/manpages-zh/archive/v1.6.3.2.tar.gz
Resolving github.com (github.com)... 192.30.253.112, 192.30.253.113
Connecting to github.com (github.com)|192.30.253.112|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-04-23 16:36:23 ERROR 404: Not Found.

!!! Couldn't download 'man-pages-zh_CN-1.6.3.2.tar.gz'. Aborting.

Reply:
I'm very sorry, I mistaken the url because I have a local file cache in my Gentoo.
This time I think I have fixed it.
And I have squashed everything in one commit.